mirrord/intproxy/Cargo.toml

```diff
@@ -37,5 +37,4 @@ hyper-util.workspace = true
 http-body-util.workspace = true
 bytes.workspace = true
 futures.workspace = true
-
```

Comment:
Don't know how I feel about this one. We might need input from the whole team for this. Should we call a meeting?